### PR TITLE
Inicjalizacja interfejsu graficznego

### PR DESCRIPTION
**Pull request by krzysjed**

### CODE REVIEW

**Review by Kamilbur — COMMENTED** [review]
Two objections:
1) gui module doesn't work for me. How do you run it?

image

2) Could we leave __ init __ files empty for vast majority of cases, as it should be. I propose making a new file f.e. main_window.py or so, just to store Window class.

**Comment by krzysjed** [comment]
I pushed fixed version. You can do review again.

File: src/gui/__main__.py [hunk 1]
```diff
@@ -1,8 +1,13 @@
+from MainWindow import Window
```
Comment (Kamilbur) [inline src/gui/__main__.py]: Still can't run it as a separate module. I belive changing this line to something more like this:
`from gui.MainWindow import Window`
would help.

**Comment by Kamilbur** [comment]
It works perfectly!

**Pull request merged**